care/emr/api/viewsets/patient.py

```
@@ -196,3 +214,42 @@
+        )
+        value = request_data.value
+        if not value and request_config.config["required"]:
+            raise ValidationError("Value is required")
+        if value:
+            try:
+                validate_identifier_config(
+                    {"config": request_config.config, "id": request_config.external_id},
+                    value,
+                )
+            except ValueError as e:
+                raise ValidationError(str(e))
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)